retrofit/build.gradle

```diff
@@ -26,3 +26,26 @@ jar {
     attributes  'Automatic-Module-Name': 'retrofit2'
   }
 }
+
+// Create a test task for each supported JDK.
+(8..17).each { majorVersion ->
```

Comment:
18 hasn't been deployed by Adoptium yet, but the 18 on the CI runner will be used by the normal 'test' task.